includes/admin/amp-backend-actions.php

```diff
@@ -0,0 +1,46 @@
+<?php
+// Callbacks for adding AMP-related things to the admin.
+
+/** AMP_Template_Customizer class */
+require_once( AMP__DIR__ . '/includes/admin/class-amp-customizer.php' );
+
+/**
+ * Instantiates the AMP template editor for the Customizer.
+ *
+ * @param WP_Customize_Manager $wp_customize Customizer instance.
+ */
+function init_amp_template_customizer( $wp_customize ) {
+	AMP_Template_Customizer::init( $wp_customize );
```

Comment:
It'd be nice to have a filter to not enable the AMP Customizer since it won't be relevant for all sites.